src/Models/Location.php

```diff
@@ -307,41 +307,41 @@ public function getDirectionsUrlAttribute()
 
     public function getBookingsYesterdayAttribute()
     {
-        return config('tipoff.model_class.booking')::yesterday()
+        return app('booking')::yesterday()
```

Comment:
@sl0wik & @pdbreen What's the best way to call a function on a resolved model?

Reply:
The answer ties back to the draft PRs and notes I circulated on Friday.  Ultimately, I think there will be 2 parts -
* service contracts and interfaces that provide API functionality (possibly w/null implementations registered by default)
* model interfaces that provide ability to interact with specific model instances that live in external packages

Reply:
Model interfaces would be the parameter types used in the service interfaces (either as inbound values or results).

Reply:
Sounds good. Let's get that nailed down in the next few days as we're working on the models. Then we can use the same style for all of the packages that need it.